### ISSUE TITLE
action.reinforce(reward)

### ISSUE THREAD

**Issue by susht3** [pytorch/examples#142]
What does "action.reinforce(reward)" mean? Does it means gradient descent?
![image](https://cloud.githubusercontent.com/assets/12723964/25036722/01b61e70-2128-11e7-9c41-3f21fb5fe13b.png)

**Comment by jekbradbury** [comment]
Yes, it's the score function estimator a.k.a. vanilla policy gradient a.k.a. REINFORCE. BTW, the forums (discuss.pytorch.org) are the best place for questions; GitHub issues are reserved for bug reports.
